internal-tooling/README.md

```diff
@@ -0,0 +1,44 @@
+# nodejs-storage benchmarking
+
+**This is not an officially supported Google product**
+
+This benchmarking script is used by Storage client library maintainers to benchmark various workloads and collect metrics in order to improve performance of the library.
```

Comment:
"This benchmarking script is used by Storage client" -> "This benchmarking script intended for use by Storage client"